docs/manual/docs/maintainer-guide/updating/index.md

```diff
@@ -1,7 +1,170 @@
-# Updating the application {#updating}
+# Updating the application
 
-Since GeoNetwork 4.0.4, passwords stored in the database for the mail server, harvesters, etc. are encrypted with [Jasypt](http://www.jasypt.org/).
+## Before you start
 
-By default, a random encryption password is generated when GeoNetwork is started, if it is not already defined, and it is stored in the file **`/geonetwork/WEB-INF/data/config/encryptor/encryptor.properties`**. If you have set the location of the data directory outside of the application, the file will be stored in this external location. Read more at [Customizing the data directory](../../install-guide/customizing-data-directory.md).
+### Jasypt encryption settings
 
-The file with the encryption settings **must be copied** to the new installation when upgrading the application; otherwise, it will not be possible to decrypt the existing passwords stored in the database.
+Since GeoNetwork 4.0.4, passwords stored in the database for the mail server, harvesters, etc. are encrypted with [Jasypt](https://www.jasypt.org/).
+    
+By default, a random encryption password is generated when GeoNetwork is started, if it is not already defined, and it is stored in the data directory **`data/config/encryptor/encryptor.properties`**.
+
+=== "Application data directory"
+
+    If you are using the application data direcotry, the encryption password is stored in the file **`/geonetwork/WEB-INF/data/config/encryptor/encryptor.properties`**. 
+    
+    !!! warning
+    
+        This **`encryptor.propertie`** file **must be copied** to the new installation when upgrading the application; otherwise, it will not be possible to decrypt the existing passwords stored in the database.
+
+=== "External data directory"
+
+    If you have set the location of the data directory outside of the application, the file will be stored in this external location at **`data/config/encryptor/encryptor.properties`**
+  
+    Read more at [Customizing the data directory](../../install-guide/customizing-data-directory.md).
+
+## Updating from GeoNetwork 4.2 to GeoNetwork 4.4
+
+Updating fro GeoNetwork 4.2 to GeoNetwork 4.4 is a minor upgrade and may be performed in place.
+
+Before you start:
+
+=== "Application data directory"
+    
+    1. Stop the services:
+        
+        * Application server running GeoNetwork
+        * Elasticserach
```

Comment:
```suggestion
```

I don't think is required to stop Elasticsearch to update GeoNetwork.

Reply:
They may need to update Elasticsearch however